### PR TITLE
Removed Font Awesome

### PR DESCRIPTION
**Pull request by fkorotkov**
Upgraded dependencies and optimized imports

Fixes #162

### CODE REVIEW

File: src/components/account/UserProfile.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ class UserProfile extends React.Component<Props> {
         variant="contained"
         href={`https://github.com/marketplace/cirrus-ci/order/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTI=?account=${user.githubUserName}`}
       >
-        <Icon className={classNames(classes.leftIcon, 'fa', 'fa-github')} />
+        <GitHubIcon className={classes.leftIcon} />
```
Comment (RDIL) [inline src/components/account/UserProfile.tsx:72]: You should use the `startIcon` prop if you are adding icons to buttons for enhanced flexibility.

File: src/components/account/UserProfile.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ class UserProfile extends React.Component<Props> {
           variant="contained"
           href={`https://github.com/marketplace/cirrus-ci/order/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTA=?account=${user.githubUserName}`}
         >
-          <Icon className={classNames(classes.leftIcon, 'fa', 'fa-github')} />
+          <GitHubIcon className={classes.leftIcon} />
```
Comment (RDIL) [inline src/components/account/UserProfile.tsx:94]: Same here.

File: package.json [hunk 3]
```diff
@@ -13,7 +13,10 @@
   "private": true,
   "license": "MIT",
   "dependencies": {
-    "@material-ui/core": "^4.9.4",
+    "@fortawesome/free-brands-svg-icons": "^5.13.0",
```
Comment (RDIL) [inline package.json:16]: Since Material-UI has a GitHub icon, we should just be able to ditch FA as a whole.

Comment (fkorotkov) [reply package.json:16]: Right, that left from my initial attempt to import these SVGs.

**Pull request merged**